main.py

```diff
@@ -71,6 +72,17 @@ def change_read():
     new_read = ui.get_read_value()     
     book.read = new_read 
     book.save()
+
+
+def delete_book():
+    book_id = ui.get_book_id()
+
+    book_deleted = store.get_book_by_id(book_id)
```

Comment:
Variable name - `book_deleted` implies the book is already deleted, which it may be, but it isn't deleted yet. How about `book_to_delete` ?